src/ExportOptions.php

```diff
@@ -16,19 +16,31 @@ class ExportOptions
      */
     private $isCompresed;
 
+    /**
+     * @var array
+     */
+    private $queryBinding;
+
     public function __construct(
         ?string $query = null,
+        array $queryBindings = [],
```

Comment:
přijde mi nebezpečný přidávat parametr doprostřed, když nevím, kde všude se to používá. Tímhle je to BC nekompatibilní změna. Takže pokud to někde někdo používá (jakože jo, jinak by to nebylo jako lib, ale bylo by to v KBC), tak doteď měl `new x(?string, bool)` a teď má `new x(?string, array)` takže tam bude typeerror,

Reply:
Oh, fuck. Tohle je vlastně ta nová! To není php-db-import 🤦‍♂